Formula/sigrok-firmware-fx2lafw.rb

```diff
@@ -0,0 +1,19 @@
+class SigrokFirmwareFx2lafw < Formula
+  desc "Open-source firmware for Cypress FX2 chips"
+  homepage "https://sigrok.org/wiki/Fx2lafw"
+  url "https://sigrok.org/download/source/sigrok-firmware-fx2lafw/sigrok-firmware-fx2lafw-0.1.7.tar.gz"
+  sha256 "a3f440d6a852a46e2c5d199fc1c8e4dacd006bc04e0d5576298ee55d056ace3b"
+
+  depends_on "make" => :build
```

Comment:
Doesn't it build with macOS make? If so, that line should be removed.

Reply:
Thanks, good pickup. I will change this and recommit it.